modules/react-native-status/ios/RCTStatus/UIHelper.h

```diff
@@ -0,0 +1,9 @@
+#import <sys/utsname.h>
+#import <Foundation/Foundation.h>
+#import <React/RCTBridgeModule.h>
+#import "Statusgo.h"
```

Comment:
safe to remove this import from here

Reply:
cleaned up here : https://github.com/status-im/status-mobile/pull/18385/commits/1cd73b0242def75538da0e0b3eee2ee18304e6ca